net/xia/ppal_ether/main.c

```diff
@@ -18,12 +20,132 @@ static inline struct xip_ether_ctx *ctx_ether(struct xip_ppal_ctx *ctx)
 		: NULL;
 }
 
+/* ETHER-INTERFACE. */
+struct ether_interface {
+	struct net_device  *dev;
+	atomic_t           refcnt;
+	struct rcu_head    rcu_head;
+
+	/* Prevents racing conditions over list. */
+	spinlock_t         interface_lock;
+	struct list_head   list_interface_common_addr;
+};
+
+static inline struct ether_interface *__ether_get_rtnl(
```

Comment:
`__ether_get_rtnl()` is not adding value and it's used only once. Could you merge it at the place where it's called?